android/src/main/java/cl/json/ShareFile.java

```diff
@@ -16,6 +16,8 @@
  */
 public class ShareFile {
 
+    public static final int BASE_64_DATA_LENGTH = 5; // `data:`
+    public static final int BASE_64_DATA_OFFSET = 16; // `;base64,` + 8
```

Comment:
curious how you tested this? I'm finding that unless this value is `8`, it leads to a corrupted file. (I'm working with a base64-encoded PDF for what it's worth)

Reply:
ah. i might have cooked it! you're right. i misread the code that i was translation (from [here](https://github.com/react-native-community/react-native-share/pull/856/files#diff-2fa2788e1b71bd42f81abefbb76a001bL109)). I don't know how this was working in my local implementation 🤔.